src/h3/_cy/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+find_package(Cython MODULE REQUIRED)
+find_package(PythonExtensions MODULE REQUIRED)
+
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+macro(add_cython_file filename)
+    add_cython_target(${filename} C PY3)
+    add_library(${filename} MODULE ${filename} ${LIB_SOURCE_FILES} ${CONFIGURED_API_HEADER})
+    python_extension_module(${filename})
+    set_property(TARGET ${filename} PROPERTY C_STANDARD 99)
+    target_compile_definitions(${filename} INTERFACE $<TARGET_PROPERTY:h3,COMPILE_DEFINITIONS>)
+    target_include_directories(${filename} INTERFACE $<TARGET_PROPERTY:h3,INCLUDE_DIRECTORIES>)
```

Comment:
I would assume these should be automatic and not needed if you change the target_link_libraries to `target_link_libraries(${filename} PUBLIC h3)`

Reply:
I have no idea how CMake works, so I just tried your suggestion. I got the following error:

<img width="844" alt="Screen Shot 2020-05-09 at 5 03 47 PM" src="https://user-images.githubusercontent.com/984039/81487902-291eaa00-9217-11ea-8ac0-aa9dbe5eb3f6.png">

I'll turn this into an issue, and we can revisit after landing this to master?

Reply:
https://github.com/uber/h3-py/issues/103